truewiki/views/source.py

```diff
@@ -72,11 +80,15 @@ def create_body(wiki_page: WikiPage, user, wrapper, preview=None, summary=None,
         "has_templates_used": "1" if templates_used else "",
         "has_used_on_pages": "1" if used_on_pages else "",
         "has_errors": "1" if errors else "",
+        "has_change_conflict": edit_conflict if edit_conflict else "",
```

Comment:
pick one: `edit_conflict` or `change_conflict`. But not both please :)

Reply:
I started with change conflict and moved to edit conflict. That slipped through my `:%s/change_/edit_/gc` I guess